voicevox_engine/metas/MetasStore.py

```diff
@@ -0,0 +1,50 @@
+import json
+from pathlib import Path
+from typing import Dict
+
+from voicevox_engine.metas.Metas import SpeakerSupportPermittedSynthesisMorphing
+
+
+class MetasStore:
+    def __init__(self, engine_speakers_path: Path) -> None:
+        self._engine_speakers_path = engine_speakers_path
+        self._loaded_metas = {
+            folder.name: self.fill_missing_speaker_properties(
+                json.loads((folder / "metas.json").read_text(encoding="utf-8"))
+            )
+            for folder in engine_speakers_path.iterdir()
+        }
+
+    @staticmethod
+    def fill_missing_speaker_properties(speaker: Dict):
+        speaker.setdefault(
+            "supported_features",
+            dict(),
+        )
+        speaker["supported_features"].setdefault(
+            "permitted_synthesis_morphing",
+            SpeakerSupportPermittedSynthesisMorphing(None),
+        )
+        return speaker
```

Comment:
`setdefault(dict())`、キーがミスってた時に予期せぬバグになるかもとちょっと思いました！
おしゃれさは減りますが実用性を加味して初期値を入力するのはどうでしょうか 👀 
```suggestion
    def fill_missing_speaker_properties(speaker: Dict):
        if "supported_features" not in speaker:
            speaker[supported_features] = {}
        if "permitted_synthesis_morphing" not in speaker["supported_features"]:
            speaker["supported_features"]["permitted_synthesis_morphing"] = (
                SpeakerSupportPermittedSynthesisMorphing(None)
            )
```

Reply:
Modelを用いるようになった関係でデフォルト値の設定をそちらを用いるように変更しました．
その影響でSpeakersの`supported_features`等のプロパティをrequiredにしたので、型定義として未対応エンジンとの不整合が生じます